scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -425,7 +429,8 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
 
                 $scope.error = {};
 
-                return checkMediaAssociatedToUpdate()
+                return functionPoints.run('authoring:publish', {orig, item})
```

Comment:
Could you only pass `item`, but not `orig`? Why do you need `orig`?

Reply:
`orig` and `item` are passed into the `publishItem` function, so I made the decision that the `FunctionPoint` should provide access to the same arguments from the parent function.

Reply:
Got it. Please don't pass `orig` to `FunctionPoint` then. You can fetch the original from the server if you ever need it.